x-pack/docs/en/security/authentication/oidc-guide.asciidoc

```diff
@@ -419,14 +419,14 @@ through either the
 NOTE: You cannot use {stack-ov}/mapping-roles.html#mapping-roles-file[role mapping files]
 to grant roles to users authenticating via OpenID Connect.
 
-This is an example of a simple role mapping that grants the `kibana_user` role
+This is an example of a simple role mapping that grants the `kibana_admin` role
 to any user who authenticates against the `oidc1` OpenID Connect realm:
 
 [source,console]
 --------------------------------------------------
 PUT /_security/role_mapping/oidc-kibana
 {
-  "roles": [ "kibana_user" ],
+  "roles": [ "kibana_admin" ],
```

Comment:
I'd be wary of suggesting to add `kibana_admin` to all users of the realm as consumers of the docs tend to follow our examples verbatim